gist/build.gradle

```diff
@@ -3,11 +3,11 @@ apply plugin: 'kotlin-android'
 apply plugin: 'maven-publish'
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
     buildToolsVersion "30.0.2"
     defaultConfig {
         minSdkVersion 19
-        targetSdkVersion 30
+        targetSdkVersion 31
```

Comment:
Ran a quick search to see what we're using in other repos, I think we will need to bump RN as well. see: https://github.com/search?q=org%3Acustomerio+targetSdkVersion&type=code